src/JustSaying/Fluent/QueueSubscriptionBuilder`1.cs

```diff
@@ -14,7 +14,7 @@ namespace JustSaying.Fluent
     /// <summary>
     /// A class representing a builder for a queue subscription. This class cannot be inherited.
     /// </summary>
-    /// <typeparam name="T">
+    /// <typeparam queueName="T">
```

Comment:
```suggestion
    /// <typeparam name="T">
```
Probably a find & replace fail here